### DISCUSSION TITLE
Regarding missing fields in telemetry documentation

### DISCUSSION THREAD

**Discussion by Ramya-kmurthy** [Sunbird-Obsrv/Community#18] [Documentation]
In the telemetry documentation these fields[Contributor Name,Vertical,Program] are not available how to get these fields.Please share the path.

CC : @rhwarrier

**Comment by pramodkvarma** [comment]
As I understand, these are indirectly mapped. If there is content/collection ID in telemetry, from content API, you can get contributor name. Not sure if these are denormalized all the time. @rhwarrier @SanthoshVasabhaktula

**Reply by SanthoshVasabhaktula** [reply]
Yes @pramodkvarma - These are denormalized within the pipeline. Location, Device, Content and User attributes are always denormalized. The denormalization fields are configurable. If any new metadata fields are added, the fields need to be added to the denorm configuration. 

@anandp504 - Please share the config file path for the content denormalizer.

**Reply by pramodkvarma** [reply]
OK great! Thanks @SanthoshVasabhaktula

**Reply by rhwarrier** [reply]
@anandp504 can you please share the config file path ?

**Reply by anandp504** [reply]
@Ramya-kmurthy  @rhwarrier  The new fields need to be added to the list of content fields that are denormalized. Please find the list of content fields [here](https://github.com/project-sunbird/sunbird-data-pipeline/blob/release-4.11.0/data-pipeline-flink/de-normalization/src/main/scala/org/sunbird/dp/denorm/task/DenormalizationConfig.scala#L51:L52).
